test/lib/cupsToMl.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../helpers');
+const lab = exports.lab = Lab.script();
+const cupsToMl = require('../../lib/cupsToMl').cupsToMl;
+const mlToCups = require('../../lib/cupsToMl').mlToCups;
+
+describe('cupsToMl', () =>  {
+  it('should convert cups to Milliliters', () => {
+    const Ml = cupsToMl(1);
```

Comment:
Lowercase for the `Ml` variable. Only classes should start with a capital letter